pkg/ctl/create/nodegroup.go

```diff
@@ -3,20 +3,28 @@ package create
 import (
 	"fmt"
 	"os"
+	"strings"
 
+	"github.com/gobwas/glob"
 	"github.com/kris-nova/logger"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 
+	"github.com/weaveworks/eksctl/pkg/ami"
 	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha4"
 	"github.com/weaveworks/eksctl/pkg/authconfigmap"
+	"github.com/weaveworks/eksctl/pkg/cfn/manager"
 	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
 	"github.com/weaveworks/eksctl/pkg/eks"
 	"github.com/weaveworks/eksctl/pkg/printers"
 	"github.com/weaveworks/eksctl/pkg/utils"
 )
 
+var (
+	nodeGroupFilter = ""
```

Comment:
could this be rather:
`nodeGroupFilter string`

which would initialize it with it's string zero-value (i.e. "")?

Reply:
It could. I just followed the pattern established by `clusterConfigFile = ""` in cluster.go.

Reply:
OK. I can go with that then...